src/Cache/Cache.php

```diff
@@ -125,7 +125,7 @@ public static function registry(ObjectRegistry $registry = null)
         if ($registry) {
             static::$_registry = $registry;
         }
-        
```

Comment:
@jippi Not my fault if people still haven't discovered the auto blank space trimming feature in their editor/ide.